src/servers/tests/mod.rs

```diff
@@ -93,13 +93,15 @@ impl ScriptHandler for DummyInstance {
         self.scripts
             .write()
             .unwrap()
-            .insert(name.to_string(), Arc::new(script));
+            .insert(format!("{schema}_{name}"), Arc::new(script));
 
         Ok(())
     }
 
-    async fn execute_script(&self, name: &str) -> Result<Output> {
-        let py_script = self.scripts.read().unwrap().get(name).unwrap().clone();
+    async fn execute_script(&self, schema: &str, name: &str) -> Result<Output> {
+        let key = format!("{schema}_{name}");
+
+        let py_script = self.scripts.read().unwrap().get(&key).unwrap().clone();
```

Comment:
Do we need to check whether the script is existed? i.e. `get(&key)` returns `None`

Reply:
Sounds reasonable!